.github/runner.bicep

```diff
@@ -0,0 +1,871 @@
+@description('The name of you Virtual Machine.')
+param utcValue string = utcNow()
+param vmName string = toLower('github-runner-${uniqueString(resourceGroup().id, utcValue)}')
+
+@description('Username for the Virtual Machine.')
+param adminUsername string = 'ubuntu'
+
+@description('Type of authentication to use on the Virtual Machine. SSH key is recommended.')
+@allowed([
+  'sshPublicKey'
```

Comment:
It shouldn't. Line 10 is listing the allowed values for this parameter, while line 12 is the current value (+default if not supplied) to be used in deployment. 

Previously the value "password" was also allowed for authentication type. I will hard-code this to sshPublicKey directly instead of providing it as parameter.